package.json

```diff
@@ -26,8 +27,14 @@
     "ws": "^7.3.1"
   },
   "devDependencies": {
+    "@types/node": "^14.14.3",
+    "@types/sqlite3": "^3.1.6",
+    "@types/ws": "^7.2.7",
+    "@typescript-eslint/eslint-plugin": "^4.6.0",
+    "@typescript-eslint/parser": "^4.6.0",
     "babel-eslint": "^10.1.0",
```

Comment:
AFAIU this can be removed as `parser` property has been changed in `.eslintrc.js`?

Reply:
I'll check it

Reply:
Removed :)